content/en/docs/ops/ambient/usage/waypoint/index.md

```diff
@@ -0,0 +1,400 @@
+---
+title: L7 - waypoint proxy Layer
+description: User guide for L7 Processing.
+weight: 2
+owner: istio/wg-networking-maintainers
+test: n/a
+---
+
+{{<tip>}}
+Before you start with this guide, please make sure that you have already read the Ztunnel Networking sub-guide and have a basic understanding of Istio Ambient Mesh. This guide assumes that you have already completed the Installation, set up the ztunnel and enabled mTLS.
+
+If you have not yet done these things, please go back to the Ztunnel Networking sub-guide and follow the instructions before proceeding with this guide.
+
+Once you have completed the Ztunnel Networking sub-guide, you will be ready to start setting up the L7 Waypoint Proxy layer in Istio Ambient Mesh.
+
+{{</tip>}}
+
+
+## Introduction 
+
+This guide provides instructions on how to set up and use the L7 Waypoint Proxy layer in Istio Ambient Mesh. It assumes that you have already read the Ztunnel Networking sub-guide and have a basic understanding of Istio Ambient Mesh.
+
+L7 traffic routing is based on L4 with the addition of the Waypoint proxy, which is more complex to handle in Envoy. We can also create HPAs to scale it dynamically.
+
+**Overview of the Layer 7 Waypoint proxy**
+
+TODO
+
+**Benefits of using the Layer 7 Waypoint proxy**
+
+TODO
+
+## Pre-requisites & Supported Topologies
+
+Before you begin, make sure that you have the following prerequisites in place:
+1. Istio Ambient Mesh installed and configured
+2. Ztunnel proxy is installed and running
+3. Mutual TLS (mTLS) enabled and configured
+
+## Understanding the L7 Waypoint Proxy Default Configuration
+
+<< Consider breaking this out into bullets for easier reading TODO >>
+
+The L7 Waypoint Proxy layer in Istio Ambient Mesh is designed to provide a usable configuration out of the box with a fixed feature set that does not require much, or any, custom configuration. Currently, there are no configuration options that need to be set other than the `waypoint` profile setting. Once this profile is used, this in turn sets two internal configuration parameters within the Istio Operator which eventually set the configuration of the L7 Waypoint Proxy.
+
+In the future, there may be some additional limited configurability for L7 Waypoint Proxies. However, for now, the following are all configured with fixed default configurations that are not customizable:
```

Comment:
Remove this text or section .. this text was appropriate for the ztunnel guide since the ztunnel proxy is designed to work with minimal user configured features and the two internal configuration parameters also refers to the ztunnel proxy. The Waypoint proxy does get configured based on resources such as Kubernetes Gateway API CRDs and Istio/VirtualService etc so this may not be as appropriate as for ztunnel.